bucket/pycharm-pro.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": "2018.3.3",
+    "homepage": "https://www.jetbrains.com/pycharm/",
+    "url": "https://download.jetbrains.com/python/pycharm-professional-2018.3.3.zip",
+    "hash": "cd9da42dcf3bf5066957df4d521a63d6b314da055ac0c382d51deeb08762eef6",
+    "env_add_path": "bin",
```

Comment:
Why this add path?? In bin folder are only internal application to help restarting, notifying, other small operations.